kong/templates/nginx_kong.lua

```diff
@@ -1,5 +1,6 @@
 return [[
 charset UTF-8;
+#server_tokens off;
```

Comment:
this is commented out here in the real template, but it is not commented in the `spec/fixtures/custom_nginx.template` below. Is this intentional?

Reply:
Oh, good catch.